src/input_cp2k_pwdft.F

```diff
@@ -161,15 +161,15 @@ SUBROUTINE fill_in_section(section, section_name)
                CALL sirius_option_get_description_usage(section_name, name, description, usage)
                SELECT CASE (ctype)
                CASE (1)
-                  CALL sirius_option_get_int(section_name, name, dummy_i, vec_length);
+                  CALL sirius_option_get_int(section_name, name, dummy_i, vec_length); 
```

Comment:
I doubt that our convention checker requires a trailing space ;-)

Reply:
Agreed, but the CI tells me that formatting is incorrect. That's why I had to run make pretty and push the changes. Emacs  suppress the trailling whitespace automatically but make pretty put it back.
I can not do much about it :(.
M.

Reply:
I will also need to have psmp regtest activated to know that everything works with the CI. This would be very useful feedback.

Reply:
Ok, I just triggered it. I've actually been contemplating to make `psmp` a required check as well. The downside is that it takes slightly longer than `sdbg`.

Reply:
My guess is that the trailing space originates from the trailing semicolon, which could simple be remove.

Reply:
I will fix that. Indeed trailing semicolon are responsible for this.